src/ONYXKEYS.js

```diff
@@ -222,6 +222,9 @@ export default {
     // Whether the auth token is valid
     IS_TOKEN_VALID: 'isTokenValid',
 
+    // The color theme setting defined by the user in preferences
+    COLOR_THEME: 'colorTheme',
```

Comment:
```suggestion
   PREFERRED_THEME: 'preferredTheme',
```

or 
 `  NVP_PREFERRED_THEME: 'preferredTheme', `